src/helpers/util.js

```diff
@@ -38,7 +38,11 @@ function uuid() {
  * @returns {*}
  */
 function promiseSeries(promises) {
-  return promises.reduce((prev, fn) => prev.then(fn), Promise.resolve());
+  return promises.reduce((prev, fn) => prev.then(data => {
+    const options = data ? { aborted: !!data.skip } : {};
+
+    return fn(options);
+  }), Promise.resolve());
```

Comment:
Please, create another method for it or move it where it is required

Reply:
Or you can add a callback parameter in this function to handle its execution. I guess it is the best option